Cargo.toml

```diff
@@ -7,6 +7,7 @@ members = [
     'node/cli',
     'node/service',
     'bin/utils/moonkey',
+    'precompiles/pallet-democracy',
```

Comment:
If the precompiles are imported by the runtime already you dont need this

Reply:
I agree we don't need it. But we have it for many of our other crates. We should have a convention for this. In Substrate they have every crate listed there. I don't know why. At some point it seemed like this was affecting whether tests were run in CI, but I'm not sure if that turned out to be right or not.

Reply:
I'm of the opinion to list all our crates in the Cargo.toml, it allows the rust-analyser to work better, and it guarantees that the CI compiles everything well.

Reply:
For now I've removed it as suggested.